src/index.html

```diff
@@ -7,11 +7,150 @@
+        <label class="line__input">
+          Name:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="line__input">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="line__input">
+          Full date of birth:
+          <input
+            type="date"
+            name="birthdate"
+          >
+        </label>
+        <label class="line__input__last">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="block_input">
+        <legend>Registration</legend>
+        <label class="line__input">
+          E-mail:
+          <input
+            type="email"
+            name="mail"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="line__input__last">
+          Password:
+          <input
+            type="password"
+            name="pass"
+            minlength="5"
+            maxlength="30"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="block_input">
+        <legend>An interesting fact about you!</legend>
+        <label class="line__input">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            value="Yes"
+          >
+          Yes
+          <input
+            type="radio"
+           name="cats"
+            value="No"
+          >
+          No
+        </label>
+        <label class="line__input">
+          What is your favorite colour?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+        <label class="line__input">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+          >
+        </label>
+        <label class="line__input">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            required
+            multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="line__inpu__last">
+          How do you rate our work?
+          <input
+            type="range"
+            name="range"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="block_input">
+        <legend>Additional info:</legend>
+        <label class="line__input">
+          Comments:
+          <textarea
+            name="text"
+            autocomplete="off"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="text"
            autocomplete="off"
            cols="20"
            rows="2"
        ></textarea>
```
![image](https://user-images.githubusercontent.com/96409563/223972236-cac8371a-5a8f-491a-b538-f7f444fe5d53.png)
![image](https://user-images.githubusercontent.com/96409563/223972291-70c9c550-4784-4a38-a250-818739182e2f.png)